libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -900,8 +915,8 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
                                     float load_factor)
 {
     // Calculate time in seconds since last update
-    uint32_t now = AP_HAL::micros();
-    _DT = MAX((now - _update_pitch_throttle_last_usec), 0U) * 1.0e-6f;
+    uint64_t now = AP_HAL::micros64();
```

Comment:
This uint32_t to uint64_t is interesting, were you concerned with it wrapping around? That MAX was fixing that but it's a hack. Is this a concern we should consider fixing globally?